packages/cfd/src/Components/__tests__/cfd-account-card.spec.tsx

```diff
@@ -0,0 +1,484 @@
+
+    const mt5_acc: TMockPlatformAccounts = {
+        account_type: 'demo',
+        balance: 10000,
+        platform: 'mt5',
+        display_balance: '10000.00',
+        display_login: '20103240',
+        landing_company_short: 'svg',
+        login: 'MTD20103240',
+        market_type: 'financial',
+    };
+
+    const derivx_acc: TMockPlatformAccounts = {
+        account_type: 'demo',
+        balance: 10000,
+        platform: 'dxtrade',
+        display_balance: '10000.00',
+        display_login: '20103240',
+        landing_company_short: 'svg',
+        login: 'MTD20103240',
+        market_type: 'financial',
+    };
+
+    const mt5_labuan_acc: TMockPlatformAccounts = {
+        account_type: 'real',
+        balance: 10000,
+        platform: 'mt5',
+        display_balance: '10000.00',
+        display_login: '20103240',
+        landing_company_short: 'labuan',
+        login: 'MTD20103240',
+        market_type: 'financial',
+    };
+
+    beforeEach(() => {
+        props = {
+            button_label: 'Top up',
+            commission_message: 'No commission',
+            descriptor: '',
+            dxtrade_tokens: {
+                demo: '',
+                real: '',
+            },
+            is_hovered: false,
+            existing_accounts_data: [],
+            has_banner: true,
+            has_cfd_account_error: false,
+            is_eu: true,
+            has_real_account: true,
+            is_accounts_switcher_on: false,
+            is_button_primary: true,
+            is_disabled: false,
+            is_logged_in: true,
+            is_virtual: true,
+            onHover: jest.fn(),
+            specs: {},
+            type: { category: '', type: '', platform: '' },
+            title: '',
+            platform: 'mt5',
+            onSelectAccount: jest.fn(),
+            onClickFund: jest.fn(),
+            onPasswordManager: jest.fn(),
+            toggleAccountsDialog: jest.fn(),
+            toggleShouldShowRealAccountsList: jest.fn(),
+            isEligibleForMoreDemoMt5Svg: jest.fn(() => true),
+            isEligibleForMoreRealMt5: jest.fn(() => true),
+            toggleMT5TradeModal: jest.fn(),
+            setMT5TradeAccount: jest.fn(),
+            trading_platform_available_accounts: [],
+        };
+    });
+
+    it('should render the component for Demo MT5 Synthetic account ', () => {
+        const type = {
+            type: 'synthetic',
+            category: 'demo',
+            platform: 'mt5',
+        };
+        render(
+            <CFDAccountCard
+                {...props}
+                type={type}
+                descriptor={synthetic_descriptor}
+                title='Synthetic'
+                existing_accounts_data={[mt5_acc]}
+            />
+        );
+        expect(screen.getByText(/most popular/i)).toBeInTheDocument();
+        expect(screen.getByText(/demo/i)).toBeInTheDocument();
+        expect(screen.getByText(/IcMt5CfdPlatform/i)).toBeInTheDocument();
```

Comment:
@thisyahlen-deriv Since I'm not familiar with CFD project, isn't `IcMt5CfdPlatform` an icon name? How can we test an icon with `getByText`?

Reply:
Im testing with the name so that if i change the platforms, i would know that the icon name also is being changed. 
So i can refer it to its name.

Reply:
@Bahar-FS perhaps it's because icons are mocked here with there own name text at line 34. 🙂 Is that right @thisyahlen-deriv ?

Reply:
Yesss 💯